CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# 1.0.10.1 (2020-11-04)
+## Edge Agent
+### Bug Fixes
+* Eliminate redundant calls to get a module's twin [286f15b](https://github.com/Azure/iotedge/commit/286f15bec93f745e2f8325908c696417d0891c5b)
+* Fix a problem with batch metrics upload [ab8de09](https://github.com/Azure/iotedge/commit/ab8de098cec64f28eba1f53fe5fbcf387add9795)
+* Fix vulnerability issues in ARM-based docker images [b8bc9f0](https://github.com/Azure/iotedge/commit/b8bc9f08a2bcc8b4ddc8bf2bf69ba6a2c8b4e87d)
+
+## Edge Hub
+### Bug Fixes
+* Fix vulnerability issues in ARM-based docker images [b8bc9f0](https://github.com/Azure/iotedge/commit/b8bc9f08a2bcc8b4ddc8bf2bf69ba6a2c8b4e87d)
+
+## iotedged
+### Bug Fixes
+* Fix DPS-X509 provisioning behind a proxy to use the client certificate for the proxied connection (0153812)[https://github.com/Azure/iotedge/commit/0153812ae1a5e4a2c0622f3a2f38d67d8102bf2b]
+* Don't fail `iotedge support-bundle` if iotedged is not running (93d1234)[https://github.com/Azure/iotedge/commit/93d1234f01e26ecd14ebb3b9fb674e458a834d7b]
+* Fix problems in the diagnostics module used by `iotedge check` (4257b87)[https://github.com/Azure/iotedge/commit/4257b87de0f4ca59eb894a5924fce743d03e24d6]
+
```

Comment:
I think
https://github.com/Azure/iotedge/commit/93d1234f01e26ecd14ebb3b9fb674e458a834d7b
should be included here as well

Reply:
+1

Reply:
It's there, line 15